scout/server/blueprints/variant/controllers.py

```diff
@@ -386,6 +386,9 @@ def clinvar_export(store, institute_id, case_name, variant_id):
 
     # gather missing transcript info from entrez (refseq id version)
     for pinned_var in pinned:
+        # Check if variant is not a previous variant that is not loaded in current case
```

Comment:
Can you rephrase without the double negative? 

Otherwise code looks good!

Reply:
Good idea!, fixed!